src/Components/List/List.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { db } from '../../lib/firebase.js';
+import { formatString } from '../../lib/helpers.js';
+import dayjs from 'dayjs';
+import './List.css';
+
+export default function List({ items, token }) {
+  const purchaseItem = (item) => {
+    const normalizedName = formatString(item.name);
+    db.collection('lists')
+      .doc(token)
+      .update({
+        [normalizedName]: {
+          name: item.name,
+          frequency: item.frequency,
+          lastPurchased: new Date(),
+        },
+      });
+  };
+
+  const isChecked = (item) => {
+    if (item.lastPurchased === null) {
+      return false;
+    } else {
+      const time = dayjs();
+      const purchasedAt = dayjs(item.lastPurchased.toDate());
+      const differenceInHours = time.diff(purchasedAt, 'h');
+
+      return differenceInHours <= 24;
```

Comment:
This is minor, but I noticed that setting the `lastPurchased` date back one day did not uncheck the box since the hours would equal 24, and it actually unchecked at 25 hours. 

```suggestion
      return differenceInHours < 24;
```

Reply:
An easy mistake to miss, thanks for the heads up! We'll commit your suggestion!